core/src/main/java/hudson/model/ListView.java

```diff
@@ -145,6 +149,9 @@ private Object readResolve() {
         }
         initColumns();
         initJobFilters();
+        if (statusFilter != null) {
+            jobFilters.add(0, new StatusFilter(statusFilter));
```

Comment:
@TobiX identified this as the problem in [JENKINS-62661]( https://issues.jenkins-ci.org/browse/JENKINS-62661): `DescribableList` does not implement `#add(int,T)`.